### DISCUSSION TITLE
Congratulations to Benthos and RedPanda

### DISCUSSION THREAD

**Discussion by tomtom215** [redpanda-data/connect#2674] [General]
Just wanted to say congrats to @Jeffail and other project contributors to Benthos.

When I first came across Benthos, I was blown away by how much he was able to accomplish with this project compared to other, much heavier solutions.

So when I came to check-in on the repo, I was redirected to redpanda-data/connect and it was obvious that RedPanda saw the same value in this project.

I wish Benthos/Connect the best in it's new chapter, and I truly hope this project keeps the spirit of open-source alive now with it's corporate leadership and allows for a truly open-core, non-crippled version to live on, which it seems like you're aiming to do over at https://github.com/redpanda-data/benthos

**Comment by Youpiiiii** [comment]
I'm new with Redpanda Kafka. It's an amazing project.
I have some questions about it. I use Helm Chart to manage Redpanda.
I saw on [https://techcrunch.com/2024/05/30/redpanda-acquires-benthos-to-expand-its-end-to-end-streaming-data-platform/?guccounter=1](https://techcrunch.com/2024/05/30/redpanda-acquires-benthos-to-expand-its-end-to-end-streaming-data-platform/?guccounter=1) a link to Redpanda connect in Redpanda console. It's a futur feature or I missed something?
Unfortunately benthos studio on [https://studio.benthos.dev/app/selfhosted](https://studio.benthos.dev/app/selfhosted) is no longer available. This's a feature that will be integrated into the project ?
I discovered [https://github.com/redpanda-data/redpanda-connect-helm-chart](https://github.com/redpanda-data/redpanda-connect-helm-chart). I imagine it's the best way to manage Redpanda connect for now? 
At the same time if anyone knows, I saw on [https://github.com/redpanda-data/console](https://github.com/redpanda-data/console) links to quota and admin in the video. How I can activate this features?

Thank you

**Reply by mihaitodor** [reply]
>  I saw on https://techcrunch.com/2024/05/30/redpanda-acquires-benthos-to-expand-its-end-to-end-streaming-data-platform/?guccounter=1 a link to Redpanda connect in Redpanda console. It's a futur feature or I missed something?
Unfortunately benthos studio on https://studio.benthos.dev/app/selfhosted is no longer available. This's a feature that will be integrated into the project ?

Yep! Please stay tuned. Also, have a look at the Redpanda Connect launch stream: https://www.youtube.com/watch?v=8gKPEoLyeIo

> I discovered https://github.com/redpanda-data/redpanda-connect-helm-chart. I imagine it's the best way to manage Redpanda connect for now?

Yes, that's a good way to deploy it in a Kubernetes cluster, although you can simply write your own deployment yaml. Connect is just a static binary.

> At the same time if anyone knows, I saw on https://github.com/redpanda-data/console links to quota and admin in the video. How I can activate this features?

Please reach out to [Redpanda Support](https://support.redpanda.com/hc/en-us) for that. It's a separate project.
